src/lib/components/Footer.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { _, locale } from 'svelte-i18n';
+
+  import { DEVS, REPO_LINK } from '$lib/const';
+  import { page } from '$app/stores'
+
+	import Links from './Links.svelte'
+
+  $: path = $page.url.pathname
+</script>
+
+<footer class="d-flex flex-column align-items-center mt-5 mx-0 py-4 px-1 px-md-5 bg-light">
+  <ul class="d-flex ms-0 ps-0 mb-0">
+    <Links {path} />
+  </ul>
+
+  <p class="mb-3">
```

Comment:
This `<p>` tag break the design on mobile:
![Screenshot from 2022-01-16 16-21-53](https://user-images.githubusercontent.com/29046067/149666232-37e4884a-12a5-4ab7-a161-c84c63ceaa4a.png)

Here an improvment with `text-align: center`
```suggestion
  <p class="mb-3 text-center">
```